dashboard/.storybook/main.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path')
+
+const toPath = (_path) => path.join(process.cwd(), '..', _path)
+
+module.exports = {
+  stories: ['../**/*.stories.mdx', '../**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  features: {
+    previewCsfV3: true,
+  },
+  webpackFinal: (config) => {
+    return {
+      ...config,
+      resolve: {
+        ...config.resolve,
+        alias: {
+          ...config.resolve.alias,
+          '@emotion/core': toPath('node_modules/@emotion/react'),
+          'emotion-theming': toPath('node_modules/@emotion/react'),
```

Comment:
Chakra uses `emotion@11` while Storybook uses `emotion@10`. Conflicting versions mean that Chakra styles won't be applied to our components. Adding these aliases forces Storybook to use v11 which avoids style conflicts.